codes/pthread_macros/pthread.m4.POSIX_BARRIER

```diff
@@ -0,0 +1,98 @@
+dnl Found on the web.  Written by Bastiaan Stougie, October 2003
+dnl Modified by Alberto Ros and Christos Sakalis, 2015
```

Comment:
Would you like to add your name here maybe? It's not necessary if you don't want to, but I thought I should ask anyway.